### ISSUE TITLE
🐞: Files does not sync properly with a MTP device

### ISSUE THREAD

**Issue by joe733** [elementary/files#1300]
## Prerequisites
- [X] I have searched open and closed issues for duplicates.

## Describe the bug
elementary Files does not synchronize properly with my Mobile (GIONEE F205 Android 7.1) via MTP protocol.

### To Reproduce
1. Connect and MTP device.
2. Create some files in the internal storage.
3. Changes not seen in elementary Files
3. Try reconnecting the device.
4. Same problem as in step 3.

### Expected behaviour
The files properly synchronize with the MTP device.

### Screenshots

| elementary Files | Files on Phone |
| --- | --- |
| ![Screenshot from 2020-04-27 10-52-09][1] | ![photo_2020-04-27_17-25-11][2] |

## Platform Information
![](https://user-images.githubusercontent.com/38415384/80301438-be568300-87c1-11ea-9eb9-39c683792718.png)

- [ ] <s>I'm using the latest version from git that I've manually compiled</s>
- [X] I'm using the latest released stable version

> Related [#914](https://github.com/elementary/files/issues/941#issuecomment-602039343)

[1]: https://user-images.githubusercontent.com/38415384/80367761-43689780-88a9-11ea-86cd-ce5f8d726eb0.png
[2]: https://user-images.githubusercontent.com/38415384/80369588-28e3ed80-88ac-11ea-9c29-29936fcaae18.jpg

**Comment by jeremypw** [comment]
I do not think it is possible to monitor directories over MTP so FIles will not detect the creation of new files on the device.  However, it should show them if you press the Refresh button on the right of the pathbar or press F5.

**Comment by joe733** [comment]
> I do not think it is possible to monitor directories over MTP
Yes, it's true.
> Files will not detect the creation of new files on the device.
No, it did no.

But, actually earlier there were two other documents in this (see picture) same folder, I deleted them (from phone) & refreshed the elementary files... and yes the were gone too. But when I added this `uml.pdf` it does not show up in the elementary files, not even after refreshing and (unplugging &) re-plugging the phone.

It's quite random, but yes frequent enough to get my attention.
